.gitignore

```diff
@@ -32,6 +32,8 @@ config/secrets.yml
 
 public/images/bg-search-form.pxm
 
+app/assets/images/favicons/*
```

Comment:
I don't think these should be gitignored. Otherwise, they'll be left out of any deployments.